vignettes/a07_updating_rsyncrosim_scripts_to_v2.Rmd

```diff
@@ -37,78 +37,107 @@ Note that when you remove a package from your library any associated datasheets
+In SyncroSim 3, the concept of addon packages no longer exists. All addon packages have either been converted to standalone packages (e.g., `burnP3PlusPrometheus`), and can be added to a library without having to load a base package first, or incorporated directly into the base packages (e.g., `stsimsf` is now part of `stsim`). Therefore, the following functions have been removed:
 
 ```{r deprecated, eval=FALSE}
 addon()
 disableAddon()
 enableAddon()
+
+datasheetRaster() # previously deprecated function. Use datasheetSpatRaster() instead.
+
+addBreakpoint()
+breakpoint()
+deleteBreakpoint()
+
+ssimUpdate()
+updatePackage()
 ```
 
 ## Library
 
-`info(ssimLibrary)` no longer shows the *Package name*, *Current package version* and *Minimum package version*, and the Input and Output folder paths have been combined into the *Data folder*.
-
-The `package` argument in the `ssimLibrary(package)` function has been renamed to `packages`
+Given this added flexibility, the `addon` argument has been removed from the `ssimLibrary()` function, and the `package` argument has been renamed to `packages`.
 
 ```{r library, eval=FALSE}
-myLibrary <- ssimLibrary(packages = c())
+# instead of using
+myLibrary <- ssimLibrary(package = "PackageName", addon = "AddonName")
+
+# use
+myLibrary <- ssimLibrary(packages = c("stsim", "helloworldSpatial"))
+```
+```{r load session run, warning = FALSE, echo = FALSE}
+# results for the section above
+library(rsyncrosim)
+mySession <- session()
+myLibrary <- ssimLibrary("Changes to rsyncrosim v2.ssim", packages=c("stsim", "helloworldSpatial"))
+scenario1 <- scenario(myLibrary, scenario = "Scenario 1")
+scenario2 <- scenario(myLibrary, scenario = "Scenario 2")
+scenario3 <- scenario(myLibrary, scenario = "Scenario 3")
+myScenario <- scenario(myLibrary, "my Scenario")
+dependency(myScenario) <- "Scenario 1"
 ```
 
-The `addon` and `template` arguments have also been removed.
+`info(ssimLibrary)` no longer shows the *Package name*, *Current package version* and *Minimum package version*, and the Input and Output folder paths have been combined into the *Data folder*. See the <a href="https://syncrosim.github.io/rsyncrosim/articles/a01_rsyncrosim_vignette_basic.html#access-model-metadata" taret="_blank">Introduction to rsyncrosim</a> for details.
+
+The `template` and `addon` arguments have also been removed. You can download template libraries from <a href="https://cloud.syncrosim.com/" target="_blank">SyncroSim Cloud</a>.
 
 ## Dependencies
 
 To view the existing dependencies for a scenario, use:
 
-```{r view dependencies, eval=FALSE}
+```{r view dependencies}
 dependency(myScenario)
 ```
 
-To set dependencies for a scenario, use:
+To set dependencies for a scenario:
 
 ```{r set dependencies, eval=FALSE}
-dependency(myScenario) <- c("Scenario 1", "Scenario 2")
-```
+# Change from:
```

Comment:
I would make the comments consistent across sections when comparing old code to new code. So, when demonstrating old code, could be:

> `# rsyncrosim version 1:`

and new code could be:

> `# rsyncrosim version 2:`